src/components/Dashboard/Dashboard.jsx

```diff
@@ -32,7 +32,7 @@ export class Dashboard extends React.Component {
                     {this.props.myWord && this.props.playedCard !== 0 && !this.props.finishedRound && <EndTurn />}
                 </div>
                 <Logout />
-                <Monster />
+                {<Monster />}
```

Comment:
What is the need for this change?

Reply:
Ah, I did what I was going to change in that line in another place so it's not needed, I'll delete it